### PR TITLE
Operator Framework Redesign

### PR DESCRIPTION
**Pull request by subhommitra**
Add changes to helm charts proposed by Marcel
Cleanup comments, etc
Update documentation

Resolves: #86

Note: This PR is on a branch where all commits have been squashed into one. Branch `operator-framework` in `subhommitra/DigitalTwin` persists individual commit messages.

### CODE REVIEW

File: helm/1-chart/charts/keycloakOperator/templates/realmjob.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        command:
+        - sh
+        - -c
+        args: 
+          - |
+            set -e;
+            access_token=$(curl -X POST ${KEYCLOAK_AUTH_SERVER}/realms/master/protocol/openid-connect/token \
+            -d "grant_type=password&client_id=admin-cli&username=${ADMIN_USERNAME}&password=${ADMIN_PASSWORD}" | \
+            sed 's/.*"access_token":"\([^"]*\)".*/\1/')
+            echo Access-Token: $access_token
```
Comment (wagmarcel) [inline helm/1-chart/charts/keycloakOperator/templates/realmjob.yaml:31]: Please remove the echo of access-token

File: helm/2-chart/charts/scorpio/templates/gateway-deployment.yaml [hunk 2]
```diff
@@ -21,6 +21,10 @@ spec:
       labels:
         service: {{ .Values.gateway.name }}
     spec:
+      initContainers:
+      - name: init-myservice
+        image: busybox:1.28
+        command: ['sh', '-c', "until wget -qO - http://eureka:8761/actuator/health| grep UP; do echo hello; sleep 1; done"]
```
Comment (wagmarcel) [inline helm/2-chart/charts/scorpio/templates/gateway-deployment.yaml:27]: replace "echo hello" by "echo waiting for eureka service"

**Review by wagmarcel — CHANGES_REQUESTED** [review]

File: helm/1-chart/charts/keycloakOperator/templates/realmjob.yaml (hunk 1 above)
Comment (subhommitra) [reply helm/1-chart/charts/keycloakOperator/templates/realmjob.yaml]: Done!

File: helm/2-chart/charts/scorpio/templates/gateway-deployment.yaml (hunk 2 above)
Comment (subhommitra) [reply helm/2-chart/charts/scorpio/templates/gateway-deployment.yaml]: Done!

**Review by wagmarcel — APPROVED** [review]

**Pull request merged**